### PR TITLE
DEV -> MASTER Release PR for initial release

### PR DESCRIPTION
**Pull request by bozmik**
New community review for initial release.

### CODE REVIEW

**Comment by apeltzer** [comment]
Please do a final bump-versions on this one, and  then we should be good to go 👍🏻 @reganhayward @bozmik

**Review by apeltzer — APPROVED** [review]
Only release bump-version required - LGTM!

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,14 +3,6 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0dev - [date]
+## 1.0.0 - 2021-02-09
```
Comment (apeltzer) [inline CHANGELOG.md:6]: No codename? ;-) https://www.codenamegenerator.com/

**Review by apeltzer — APPROVED** [review]
Looks good to me :-)

**Comment by apeltzer** [comment]
@MaxUlysse is busy giving an nf-core/bytesize talk today, but someone from @nf-core/core might want to give this a shot :-)

**Comment by MaxUlysse** [comment]
yeah, sorry about that, I'll promise I'll have a look after this afternoon if it's not approved already

File: CHANGELOG.md (hunk 1 above)
Comment (reganhayward) [reply CHANGELOG.md]: Good thinking - updated :-)

**Comment by reganhayward** [comment]
> yeah, sorry about that, I'll promise I'll have a look after this afternoon if it's not approved already

File: conf/test.config [hunk 2]
```diff
@@ -13,56 +13,42 @@ params {
   
   // Limit resources so that this can run on GitHub Actions
   max_cpus = 2
-  max_memory = 6.GB
+  max_memory = 4.GB
```
Comment (MaxUlysse) [inline conf/test.config:16]: Why 4GB and not 6?

File: .github/workflows/awsfulltest.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+          AWS_JOB_DEFINITION: ${{ secrets.AWS_JOB_DEFINITION }}
+          AWS_JOB_QUEUE: ${{ secrets.AWS_JOB_QUEUE }}
+          AWS_S3_BUCKET: ${{ secrets.AWS_S3_BUCKET }}
+        run: |
+          aws batch submit-job \
+            --region eu-west-1 \
+            --job-name nf-core-dualrnaseq \
+            --job-queue $AWS_JOB_QUEUE \
+            --job-definition $AWS_JOB_DEFINITION \
+            --container-overrides '{"command": ["nf-core/dualrnaseq", "-r '"${GITHUB_SHA}"' -profile test --outdir s3://'"${AWS_S3_BUCKET}"'/dualrnaseq/results-'"${GITHUB_SHA}"' -w s3://'"${AWS_S3_BUCKET}"'/dualrnaseq/work-'"${GITHUB_SHA}"' -with-tower"], "environment": [{"name": "TOWER_ACCESS_TOKEN", "value": "'"$TOWER_ACCESS_TOKEN"'"}]}'
```
Comment (MaxUlysse) [inline .github/workflows/awsfulltest.yml:43]: Why not using the test_full profile here?

**Comment by MaxUlysse** [comment]
Everything seems in order.
I apologize for the late review.

I really have only one question, why not using the `test_full` profile for the awsfulltest?

File: conf/test.config (hunk 2 above)
Comment (bozmik) [reply conf/test.config:16]: Actually, there was no particular reason for that. I will set it to 6 back :)

Comment (reganhayward) [reply conf/test.config]: That was left over from testing on my virtual machine with only 4GB of memory :-)

File: .github/workflows/awsfulltest.yml (hunk 3 above)
Comment (reganhayward) [reply .github/workflows/awsfulltest.yml]: An oversight - have updated!

**Review by MaxUlysse — APPROVED** [review]
LGTM

**Pull request merged**